docs/services/thorlabs_fw102c.rst

```diff
@@ -1,14 +1,38 @@
 Thorlabs Filter Wheel
 =====================
+This service operates a Thorlabs filter wheel.
+
+Successfully tested with the following devices:
+
+- `ThorLabs FW 102C <https://www.thorlabs.com/thorproduct.cfm?partnumber=FW102C#ad-image-0>`_
 
 Configuration
 -------------
+.. code-block:: YAML
+
+    filter_wheel:
+        service_type: thorlabs_fw102c
+        simulated_service_type: thorlabs_fw102c_sim
+        requires_safety: false
+
+        visa_id: ASRL6::INSTR
+
+        filters:
```

Comment:
Are these positions resolved by their name in the fw proxy as well? If so, might be worth adding the same comment in here (`# named positions resolved by the proxy.`) like in the example service config of the flip mount.

Reply:
good idea! added